### PR TITLE
Translation system + help command

### LINKED ISSUE #69: Add a help command

### LINKED ISSUE #96: Money command shows the user's name even when they're the one running it.
When you type `s!money` (without arguments), it says "User#0000 has X Switchcoins". I think it would be nicer if it said "You have X Switchcoins" instead.

### PR DESCRIPTION
**Pull request by pedrofracassi**
## TO-DOs
- [x] Make all commands translatable using `i18next`
- [x] Add `crowdin-node` as an API
- [x] Download locales from Crowdin before loading them into i18next
- [x] Add the help command
- [x] Add language changer command

## Environment variables
- `CROWDIN_API_KEY` - Crowdin API key
- `CROWDIN_PROJECT_ID` - Crowdin project ID (`switchblade` in our case) 

## Issues closed
- Closes #96
- Closes #69

### CODE REVIEW

File: src/commands/memes/owo.js [hunk 1]
```diff
@@ -9,16 +9,16 @@ module.exports = class OwO extends Command {
...
     const embed = new SwitchbladeEmbed(message.author)
     message.channel.startTyping()
     if (args.length > 0) {
       const { body } = await snekfetch.get(OWOapi + args.join(' '))
       embed.setTitle(body.owo)
     } else {
       embed.setColor(Constants.ERROR_COLOR)
-        .setTitle('You need to give me a sentence to owoify')
-        .setDescription(`**Usage:** ${process.env.PREFIX}${this.name} <sentence>`)
+        .setTitle(translation('commands:owo.missingSentence'))
```
Comment (davipatury) [inline src/commands/memes/owo.js]: `t`

File: src/structures/Command.js [hunk 2]
```diff
@@ -32,8 +32,8 @@ module.exports = class Command {
    * @param {Message} message Message that triggered it
    * @param {Array<string>} args Command arguments
    */
-  async _run (message, args) {
-    return this.run(message, args)
+  async _run (message, args, translation) {
+    return this.run(message, args, translation)
```
Comment (davipatury) [inline src/structures/Command.js]: `t`

File: src/structures/Command.js [hunk 3]
```diff
@@ -32,8 +32,8 @@ module.exports = class Command {
    * @param {Message} message Message that triggered it
    * @param {Array<string>} args Command arguments
    */
-  async _run (message, args) {
-    return this.run(message, args)
+  async _run (message, args, translation) {
```
Comment (davipatury) [inline src/structures/Command.js]: `t`

File: src/commands/misc/invite.js [hunk 4]
```diff
@@ -7,13 +7,13 @@ module.exports = class Invite extends Command {
     this.aliases = []
   }
 
-  async run (message) {
+  async run (message, args, translation) {
```
Comment (davipatury) [inline src/commands/misc/invite.js]: `t`

File: src/commands/misc/invite.js [hunk 5]
```diff
@@ -7,13 +7,13 @@ module.exports = class Invite extends Command {
...
 
-  async run (message) {
+  async run (message, args, translation) {
     message.channel.startTyping()
     const invite = await this.client.generateInvite()
     message.channel.send(
       new SwitchbladeEmbed()
         .setThumbnail(this.client.user.displayAvatarURL)
-        .setDescription(`[Click here to invite me to your server](${invite})\nNote that you need the \`MANAGE_SERVER\` permission to add bots to servers.`)
+        .setDescription(`[${translation('commands:invite.clickHere')}](${invite})\n${translation('commands:invite.noteThat')}`)
```
Comment (davipatury) [inline src/commands/misc/invite.js]: `t`

**Comment by pedrofracassi** [comment]
Ready for review

File: src/commands/misc/avatar.js [hunk 6]
```diff
@@ -7,12 +7,12 @@ module.exports = class Avatar extends Command {
...
   }
 
-  run (message) {
+  run (message, args, t) {
     const user = message.mentions.users.first() || message.author
     message.channel.send(
       new SwitchbladeEmbed(message.author)
         .setImage(user.displayAvatarURL)
-        .setDescription(user + '\'s avatar')
+        .setDescription(t('commands:avatar.someonesAvatar', {user: user}))
```
Comment (davipatury) [inline src/commands/misc/avatar.js]: instead of `{user: user}` you can use just `{user}`

File: src/commands/memes/owo.js [hunk 7]
```diff
@@ -9,7 +9,7 @@ module.exports = class OwO extends Command {
     this.aliases = ['uwu', 'whatsthis', 'owoify']
   }
 
-  async run (message, args) {
+  async run (message, args, t) {
```
Comment (davipatury) [inline src/commands/memes/owo.js]: missing strings maybe?

File: src/apis/Crowdin.js [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+    super()
+    this.name = 'crowdin'
+    this.envVars = ['CROWDIN_API_KEY', 'CROWDIN_PROJECT_ID']
+  }
+
+  load () {
+    return new Crowdin({
+      apiKey: process.env.CROWDIN_API_KEY,
+      endpointUrl: 'https://api.crowdin.net/api/project/' + process.env.CROWDIN_PROJECT_ID
+    });
```
Comment (bolsomito) [inline src/apis/Crowdin.js]: semicolon, `npm run fix`

Comment (bolsomito) [reply src/apis/Crowdin.js]: nvm, i'm fixing it

**Comment by pedrofracassi** [comment]
About the [vulnerability](https://snyk.io/org/pedrofracassi/test/github/bebc5043-a894-4a17-86e0-4fd079ff2ee1/master..5042b8e4b36d65cccdbf87eb6ce1c5f610930b78?fromStatus=true#npm:lodash:20180130): I don't think it's valid for us, as the user can't pass anything to lodash

File: src/commands/memes/owo.js (hunk 7 above)
Comment (pedrofracassi) [reply src/commands/memes/owo.js]: fixed.

**Comment by xDoges** [comment]
webhook test, ignore that.

**Comment by Doges** [comment]
testing again, ignore it.

**Comment by pedrofracassi** [comment]
We're almost there!

**Comment by bolsomito** [comment]
`SwitchbladePlayerManager` pls

**Comment by pedrofracassi** [comment]
@bolsomito what do you mean?

**Comment by bolsomito** [comment]
look at Travis on #development-logs and you'll see what I mean

**Comment by bolsomito** [comment]
oh, and standard aswell

**Comment by pedrofracassi** [comment]
Fixed both problems already. Will push in a sec.

**Comment by pedrofracassi** [comment]
# Ready for review!

**Comment by Doges** [comment]
v u l n e r a b l e

**Comment by pedrofracassi** [comment]
@Doges It's a lodash issue caused by strings entered by users. I don't think it affects us since lodash is being used by the crowdin API.

**Pull request merged**